admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/DiConfig.kt

```diff
@@ -78,9 +82,10 @@ fun DI.Builder.userServicesConfig(userRepoType: UserRepoType) {
 
 fun DI.Builder.userRepositoriesConfig(userRepoType: UserRepoType) {
     bind<UserRepository>() with singleton {
+        logger.info { "The user repository type is $userRepoType" }
         when (userRepoType) {
             DB ->  UserRepositoryImpl()
-            UserRepoType.ENV -> EnvUserRepository(
+            ENV -> EnvUserRepository(
                 env = instance<Application>().environment.config,
                 passwordService = instance()
             )
```

Comment:
- will type system automatically handle errors in configuration (e.g. "Db", "db", "env", "somerandomstring")
- or we should add `else` clause to check that?

Reply:
yes